src/mesh/vtk_helper.jl

```diff
@@ -16,6 +15,11 @@ function n_verts_between(n, from, to)
     return edge_verts
 end
 
+function sort_by_axis(corner_verts)
+    permutation = sortperm([corner_verts[:,i] for i in 1:size(corner_verts)[2]])
```

Comment:
```suggestion
    permutation = sortperm([corner_verts[:, i] for i in 1:size(corner_verts, 2)])
```
Can you also add a comment on what this function does?